### PR TITLE
Docs: Update visualization topic and template

### PR DESCRIPTION
**Pull request by imatwawana**
Updates Visualization topic under Structure section as well was template

To be merged after [PR 91220](https://github.com/grafana/grafana/pull/91220).

### CODE REVIEW

File: docs/sources/structure/topic-types/visualization/index.md
Comment (imatwawana) [inline docs/sources/structure/topic-types/visualization/index.md:102]:
```suggestion
<!-- vale Grafana.Gerunds = NO -->
## Naming conventions
<!-- vale Grafana.Gerunds = YES -->
```

File: docs/sources/structure/topic-types/visualization/index.md
Comment (imatwawana) [inline docs/sources/structure/topic-types/visualization/index.md:101]:
```suggestion

<!-- vale Grafana.Gerunds = NO -->
```

**Review by jdbaldry — APPROVED** [review]
Nice, thanks for updating this. Would you like to be CODEOWNER on the file so you can review any future changes?

**Comment by imatwawana** [comment]
> Nice, thanks for updating this. Would you like to be CODEOWNER on the file so you can review any future changes?

Sure, I think that makes sense! 

Also, I'll merge this once the updated time series page is merged so the example links make sense.

**Review by chri2547 — APPROVED** [review]

**Pull request merged**
